src/api/axios.api.ts

```diff
@@ -0,0 +1,7 @@
+// http://localhost:8001/api/{NeededService}
```

Comment:
This whole logoc should be moved to auth-service.ts

Reply:
Done